config/mapping-rules.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "mqttTopic": "building/{building}/room/{room}/#",
+    "kafkaTopic": "building_{building}_room_{room}"
+  },
+  {
+    "mqttTopic": "sensors/+/data",
+    "kafkaTopic": "sensor_data"
+  },
+  {
+    "mqttTopic": "devices/{device}/data",
+    "kafkaTopic": "devices_{device}_data"
+  },
+  {
+    "mqttTopic": "fleet/{fleet}/vehicle/{vehicle}/#",
+    "kafkaTopic": "fleet_{vehicle}"
+  },
+  {
+    "mqttTopic": "building/{building}/#",
+    "kafkaTopic": "building_{building}_others"
+  },
+  {
+    "mqttTopic": "sensors/#",
+    "kafkaTopic": "sensor_others"
+  },
+  {
+    "mqttTopic": "building/#",
+    "kafkaTopic": "building_others"
+  }
+]
```

Comment:
we don't need this here, because it's a user specific file. While the application.properties file can have some defaults which we are setting, the rules is business/use case specific. We can provide an example somewhere but not in this folder where we expect something more "official".

Reply:
@ppatierno what about the mapping rules for testing?

Reply:
mapping rules for testing are ok, they can stay there.